.buildkite/pipeline.yml

```diff
@@ -1,3 +1,66 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  SETUP_GVM_VERSION: 'v0.5.0'
+  DOCKER_COMPOSE_VERSION: '1.25.5'
```

Comment:
are these the versions used currently for gvm and docker-compose ?

Reply:
The value of the variable SETUP_GVM_VERSION I used from https://github.com/elastic/elastic-package/blob/main/.buildkite/pipeline.yml because I didn't find it in the fleet-server repo.
The value of the variable DOCKER_COMPOSE_VERSION I used from https://github.com/elastic/fleet-server/blob/main/.ci/Jenkinsfile
If it needs to change, please let share with me the new values of these variables.